### PR TITLE
Report Host OS version to the API

### PR DESCRIPTION
**Pull request by pcarranzav**
connects to #75

[Review on Reviewable](https://reviewable.io/reviews/resin-io/resin-supervisor/82)

### CODE REVIEW

File: src/device.coffee [hunk 1]
```diff
@@ -177,3 +177,17 @@ exports.updateState = do ->
...
+	fs.readFileAsync(path)
+	.then (releaseData) ->
+		lines = _.split(releaseData, "\n")
+		releaseItems = {}
+		for line in lines
+			[ key, val ] = _.split(line, '=')
+			releaseItems[_.trim(key)] = _.trim(val)
+		return releaseItems['PRETTY_NAME']
+	.catch (err) ->
+		console.log("Could not get OS Version: ", err)
```
Comment (Page-) [inline src/device.coffee]: Can you add an `err.stack`

File: src/device.coffee [hunk 2]
```diff
@@ -177,3 +177,17 @@ exports.updateState = do ->
 		if !applyPromise.isPending()
 			applyState()
 		return
+
+exports.getOSVersion = ->
+	path = '/mnt/root/etc/os-release'
```
Comment (Page-) [inline src/device.coffee]: Can this be in config.coffee instead

Comment (pcarranzav) [inline src/device.coffee]: You mean the OS version path or the whole function?

Comment (Page-) [inline src/device.coffee]: The path

File: src/config.coffee [hunk 3]
```diff
@@ -36,3 +36,4 @@ module.exports = config =
 		logsChannel: process.env.RESIN_SUPERVISOR_LOGS_CHANNEL ? null
 	vpnStatusPath: process.env.VPN_STATUS_PATH ? '/mnt/root/run/openvpn/vpn_status'
 	checkInt: checkInt
+	hostOsVersionPath: process.env.HOST_OS_VERSION_PATH ? '/mnt/root/etc/os-release'
```
Comment (Page-) [inline src/config.coffee]: This is missing a newline at the end

**Comment by Page-** [comment]
LGTM

**Comment by Page-** [comment]
Reviewed 2 of 3 files at r1, 1 of 2 files at r2, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/resin-io/resin-supervisor/82)*

**Comment by Page-** [comment]
Reviewed 1 of 1 files at r4.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/resin-io/resin-supervisor/82)*

**Pull request merged**